### ISSUE TITLE
Jupyterlab Terminal reporting errors on start

### ISSUE THREAD

**Issue by costrouc** [Quansight/qhub#542]
## Describe the bug

When starting a new jupyterlab session the following is being reported. Though I can report that umask is working!

```shell
/bin/sh: 16: /etc/profile.d/term_settings.sh: shopt: not found
/bin/sh: 24: /etc/profile.d/term_settings.sh: shopt: not found
/bin/sh: 58: /etc/profile.d/term_settings.sh: Syntax error: Bad fd number
```

cc: @leej3

**Comment by leej3** [comment]
Neat. I'm glad that is now water under the bridge!

 I wonder why it's using /bin/sh now. Setting the details of the terminado startup are described here: https://github.com/jupyterlab/jupyterlab/issues/3094

**Comment by costrouc** [comment]
Yeah I was wondering why `/bin/bash` was not the default. We definitely want it to be.

**Comment by leej3** [comment]
My guess, is that previously the CMD directive was previously explicitly set which I removed. We can chat about it to figure out all the various settings are interacting.

**Comment by leej3** [comment]
Resolved in 0.3.7, thanks @costrouc
